platform/net/routes_searcher_interface.go

```diff
@@ -8,11 +8,15 @@ type Route struct {
 }
 
 type RoutesSearcher interface {
-	SearchRoutes() ([]Route, error)
+	SearchRoutes(ipv6 bool) ([]Route, error)
```

Comment:
can we choose `ipv6`, `isIpv6` (or perhaps `isIPv6`)... 

`is_ipv6` seems wrong in that Golang suggest camelCase for variables, as does `IsVersion6` (see `func (ia failingInterfaceAddress) GetIP(IsVersion6 bool) (string, error)`)